lib/generate-id.js

```diff
@@ -21,12 +20,47 @@ const findId = async ( model, length ) => {
   return id
 }
 
+const idTypes = {
+  shabad: [ Shabads, 3 ],
+  line: [ Lines, 4 ],
+}
+
 /**
- * Generates a line id and a shabad id, using the database.
- */
+  * Logs the id to the console
+  * @param {Number} count the count of id to generate
+  * @param {String} type check `availableTypes`
+  */
+const logGenerateIds = async ( count, type ) => {
+  const newId = idTypes[ type ]
+
+  // Grab ids to exclude from database
+  const ids = new Set( ( await newId[ 0 ].query().select( 'id' ) ).map( ( { id } ) => id ) )
+
+  // Loop to print id's
+  for ( let i = 0; i < count; i += 1 ) {
+    console.log( `${await findId( ids, newId[ 1 ] )}` )
```

Comment:
YOu also need to update the set with the new ID when you find it, otherwise you may end up generating duplicates, so somewhere `ids.add( newId )`